packages/core/src/fhir-deduplication/resources/condition.ts

```diff
@@ -90,7 +91,7 @@ export function groupSameConditions(conditions: Condition[]): {
       const display = extractDisplayFromConcept(condition.code);
       if (display) {
         const compKey = JSON.stringify({ display, date });
-        fillMaps(dispayMap, compKey, condition, refReplacementMap, undefined, removeOtherCodes);
+        fillMaps(displayMap, compKey, condition, refReplacementMap, undefined, c => c);
```

Comment:
we are currently removing the very code we used as the key!

Reply:
Do we need the c=>c? Can't we just not pass anything and have the same result?